PLUGINS.md

```diff
@@ -412,6 +412,7 @@ Content of `state` (updated to the current journal entry):
 | `ModulesValue` |            `int`            | Value of the current ship's modules                                                                             |
 | `Rebuy`        |            `int`            | Current ship's rebuy cost                                                                                       |
 | `Modules`      |           `dict`            | Currently fitted modules                                                                                        |
+| `CargoJSON`    |           `dict`            | content of cargo.json as of last read.                                                                          |
```

Comment:
Shift this up to just under the 'Cargo' one, and call out in the 'Cargo' one that it's the *totals*.